From 527bc37dca85811bcd881b7da10e47331fb4ae5c Mon Sep 17 00:00:00 2001
Message-Id: <527bc37dca85811bcd881b7da10e47331fb4ae5c.1453151886.git.jen@redhat.com>
In-Reply-To: <e445c6fbfa4c35276b2869057fdcd8e546d63a4c.1453151886.git.jen@redhat.com>
References: <e445c6fbfa4c35276b2869057fdcd8e546d63a4c.1453151886.git.jen@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Thu, 14 Jan 2016 08:52:41 -0500
Subject: [CHANGE 6/8] qemu-options: Fix texinfo markup
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1452761561-17633-2-git-send-email-armbru@redhat.com>
Patchwork-id: 68751
O-Subject: [RHEL-6.8 qemu-kvm PATCH] qemu-options: Fix texinfo markup
Bugzilla: 1250442
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>

Fixed upstream in commit 3dbf2c7.  We should've backported it along
with commit ed24cfa, but we committed an independent rewrite instead
(commit 3dd83c0), and missed the fix.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Jeff E. Nelson <jen@redhat.com>
---
 qemu-options.hx | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/qemu-options.hx b/qemu-options.hx
index e5592ee..3b7122a 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -2203,10 +2203,6 @@ Normally QEMU loads a configuration file from @var{sysconfdir}/qemu.conf and
 option will prevent QEMU from loading these configuration files at startup.
 ETEXI
 
-STEXI
-@end table
-ETEXI
-
 #if defined(TARGET_SPARC) || defined(TARGET_PPC)
 DEF("prom-env", HAS_ARG, QEMU_OPTION_prom_env,
     "-prom-env variable=value\n"
@@ -2302,3 +2298,8 @@ STEXI
 Dump json-encoded vmstate information for current machine type to file
 in @var{file}
 ETEXI
+
+HXCOMM This is the last statement. Insert new options before this line!
+STEXI
+@end table
+ETEXI
-- 
2.1.0

